spec/fixtures/categories.yml

```
@@ -0,0 +1,47 @@
+# == Schema Information
+# Schema version: 20231110122337
+#
+# Table name: categories
+#
+#  id           :bigint           not null, primary key
+#  parent_id    :integer
+#  lft          :integer          not null
+#  rgt          :integer          not null
+#  category_tag :string
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  title        :string
+#  description  :string
+#
+
+public_body_root_category:
+  id: 1
+  lft: 0
+  rgt: 9
+
+useless_lonely_category:
+  id: 2
+  parent_id: 1
+  lft: 1
+  rgt: 8
+
+useless_category:
+  id: 3
+  parent_id: 2
+  lft: 2
+  rgt: 3
+  category_tag: useless_agency useless
```

This matches the `spec/fixtures/public_body_categories.yml` but should it just be:

```suggestion
  category_tag: useless_agency
```

---

Yeah I don't think the current system supports finding from multiple tags yet. That category would create the URL `/body/list/useless_agency%20useless` which won't find anything.